cluster-up/cluster/kind-1.19-sriov/provider.sh

```diff
@@ -18,9 +18,9 @@ fi
 SRIOV_TESTS_NS="${SRIOV_TESTS_NS:-kubevirt-test-default1}"
 
 function set_kind_params() {
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-quay.io/kubevirtci/kindest_node:v1.17.0}"
-    export KIND_VERSION="${KIND_VERSION:-0.7.0}"
-    export KUBECTL_PATH="${KUBECTL_PATH:-/kind/bin/kubectl}"
+    export KIND_VERSION="${KIND_VERSION:-0.11.1}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-quay.io/kubevirtci/kindest_node:v1.19.11@sha256:cbecc517bfad65e368cd7975d1e8a4f558d91160c051d0b1d10ff81488f5fb06}"
```

Comment:
@fgimenez
can we push it such it will have the same hash as the one appears here ?
https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1

1.19: kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729

I understand its the same image, just wonder if we can push it such it will be also the same hash ?
just for order

Thanks

Reply:
I think its not possible to set fixed hash as far as I know it is auto-generated
If there are no restrictions on the tag format, we could change it from `v1.19.11` to `v1.19.11-07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729`

Reply:
i mean that on https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1
we need to use `1.19: kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729`
which is what you used (for 1.19 use the one supplied)
the hash you used is different just because it was pushed manually to quay (i downloaded them both and verified)
Federico shown me one time that he had a way to mirror the image such that the hash will be kept (maybe the hash should have same name on both repository for), this is what i am referring to
its a nice to have, but will fit the code to the hashes appears on 
https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1

this what we did also when mirroring some other images from docker.io to quay,io
kept the hashes for better correlation




there are no restrictions, but no need to set it to such long name

Reply:
Yes, in this case it doesn't seem to be possible to keep the same digest, maybe the change in the repository name is involved:
```
$ docker tag kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729 quay.io/kubevirtci/kindest_node:v1.19.11 
$ docker push quay.io/kubevirtci/kindest_node:v1.19.11 
The push refers to repository [quay.io/kubevirtci/kindest_node]
621c6ee110a6: Layer already exists 
99793c9223da: Layer already exists 
c3433c982ca6: Layer already exists 
a1e46c95135f: Layer already exists 
7f6e587cd728: Layer already exists 
904e4a9a0839: Layer already exists 
94a557da6f32: Layer already exists 
27e8e8548e74: Layer already exists 
d1123620da17: Layer already exists 
59a7b5e0e987: Layer already exists 
a4711bd62ffb: Layer already exists 
1831449c5806: Layer already exists 
v1.19.11: digest: sha256:cbecc517bfad65e368cd7975d1e8a4f558d91160c051d0b1d10ff81488f5fb06 size: 2821
```

Reply:
Thanks

Reply:
The registry information is indeed in the image config so the digest will change when switching registries. This is also why side-loading an by digest doesn't work as expected.

https://github.com/moby/moby/blob/master/image/spec/v1.2.md